src/Controller/AddWeatherNotificationController.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Controller;
+
+use App\Message\SendNotificationDefinitionConfirmation;
+use App\Repository\NotificationDefinitionRepository;
+use App\Repository\UserRepository;
+use App\Response\ApiResponse;
+use App\Response\BadRequestApiResponse;
+use App\Response\SuccessApiResponse;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Messenger\MessageBusInterface;
+use Symfony\Component\Routing\Annotation\Route;
+
+class AddWeatherNotificationController extends AbstractController
```

Comment:
To w sumie powinno zawierać w nazwie Definition. Sugeruje coś typu `AddNotificationDefinitionController`. Przeglądnij inne miejsca czy się dobrze nazywają - anlogicznie do tej poprawki